x-pack/plugins/fleet/public/applications/fleet/sections/data_stream/list_page/index.tsx

```diff
@@ -202,6 +202,7 @@ export const DataStreamListPage: React.FunctionComponent<{}> = () => {
     <EuiInMemoryTable
       loading={isLoading}
       hasActions={true}
+      tableLayout="auto"
```

Comment:
this line improves wrapping on data streams table - makes it less likely to wrap.
8. Text in the table should wrap correctly
![image](https://user-images.githubusercontent.com/90178898/132849347-95927765-44a3-402d-bbc7-d4b44f29d6a5.png)